drift/lib/src/runtime/query_builder/components/order_by.dart

```diff
@@ -14,6 +14,20 @@ const _modeToString = {
   OrderingMode.desc: 'DESC',
 };
 
+/// Describes how to order nulls
+enum NullsOrder {
+  /// Place NULLs at the start
+  first,
+
+  /// Place NULLs at the end
+  last,
+}
+
+const _nullsOrderToString = {
+  NullsOrder.first: 'NULLS FIRST',
+  NullsOrder.last: 'NULLS LAST',
+};
```

Comment:
Now that we're targeting Dart 2.17, you can actually move this information into the enum:

```dart
enum NullsOrder {
  first._('NULLS FIRST'),
  last._('NULLS LAST');

  final String _sql;
  const NullsOrder._(this._sql);
}
```

Reply:
Yeah, I was thinking about it, but stay with one code-style.